### ISSUE TITLE
Unit test failures in 0.16.2

### ISSUE THREAD

**Issue by ddkinnamon** [arq5x/gemini#508]
Hi,

Running the unit test script on 0.16.2 results in a number of failures. I traced many of these to the recent decision to make installing tabix/bgzip/bedtools the responsibility of the end-user and so was able to fix them by adding the paths to my installs to PATH. However, it would be helpful to add this fact to the documentation on running the unit tests so that other users don't have to poke around to figure out the problem.

After making the appropriate modifications to PATH, the very last unit test still fails to run with the following message:
```
Traceback (most recent call last):
  File "test-comphet.py", line 5, in <module>
    from gemini import tests
ImportError: No module named gemini
```
Please advise.

Thanks,
Dan

**Comment by brentp** [comment]
Hi @ddkinnamon thanks again for reporting.
I made those tests check to see if the appropriate executable was available. I'll merge into master shortly.
I also slightly modified the last test for python. I saw that when I was developing it, but I am now not able to reproduce so it may be fixed in master. Either way, your install should still work.

**Comment by mjsduncan** [comment]
hi brent, fyi with git head 51d9db4 i am getting this similar error:

```
Traceback (most recent call last):
  File "test-comphet.py", line 5, in <module>
    from gemini import tests
ImportError: cannot import name tests
```

but no other errors (except for my strange never resolved failures of cadd.t01 and gerp.t02 from issue #456)

mike d

**Comment by arq5x** [comment]
Tests that use gemini_python will only work if you installed Gemini with the automated installer.

**Comment by ddkinnamon** [comment]
I did.

**Comment by brentp** [comment]
Is your system using modules?

**Comment by ddkinnamon** [comment]
Good hunch, but I think I may have found the problem in the code. It turns out that older commits of `master-test.sh` call `python test-comphet.py`, not `gemini_python test-comphet.py`. The commit for release 0.16.2 is one of these. That would explain why all the other tests work for me and why you can no longer reproduce the bug.

**Comment by brentp** [comment]
does gemini_python work for you? both gemini_python and python test-comphet.py work for me.

**Comment by ddkinnamon** [comment]
It works fine with gemini_python, but not with plain python.

**Comment by brentp** [comment]
@ddkinnamon can we close this? I think forcing update to 0.16.+ is good since we fixed a lot of issues from 0.15

**Comment by ddkinnamon** [comment]
It looks like these are all fixed in 0.16.3, which I am going to install right now...
